src/PrestaShopBundle/Resources/config/services/core/grid/doctrine_query_builder.yml

```diff
@@ -206,7 +206,7 @@ services:
             - '%database_prefix%'
             - '@prestashop.core.query.doctrine_search_criteria_applicator'
             - '@=service("prestashop.adapter.legacy.context").getContext().language.id'
-            - '@=service("prestashop.adapter.shop.context").getContextListShopID()'
+            - '@=service("prestashop.adapter.shop.context").getContextListShopID("share_order")'
 
```

Comment:
Here "share_order" is the value of a constant in the Shop ObjectModel: `const SHARE_ORDER = 'share_order';`

I'm sure there must be a way not to hardcode it in the yml but I don't know how 😅  Any idea ?

Reply:
You can have a look at https://symfony.com/blog/new-in-symfony-3-2-php-constants-in-yaml-files :)

Reply:
thanx !

Reply:
@PierreRambaud 

Unfortunately I cannot use this directly in a method parameter.

I also tried putting the class constant in a yaml parameter, and using it as a parameter for the getContextListShopID method, it doesn't work either...

I see no other way than to hardcode it for now...

Reply:
@PierreRambaud ok with @matks we found that we have to use the "parameter()" method 👍